devtools/maven/src/main/java/io/quarkus/maven/QuarkusProjectStateMojoBase.java

```diff
@@ -50,11 +50,20 @@ public void doExecute(QuarkusProject quarkusProject, MessageWriter log) throws M
             throw new MojoExecutionException("This goal requires a Quarkus extension registry client to be enabled");
         }
 
+        // to support profiles
+        final String emb = System.getProperty("quarkus.bootstrap.effective-model-builder");
+        System.setProperty("quarkus.bootstrap.effective-model-builder", "true");
+
         final Collection<Path> createdDirs = ensureResolvable(new DefaultArtifact(project.getGroupId(),
```

Comment:
AFAICS, this can also throw an exception. In that case we would leave the system property behind.
Probably not a big thing..

Reply:
Right, thanks for the comment.